src/v2.rs

```diff
@@ -416,12 +421,12 @@ impl FromGraphNpmPackages {
     specifier: impl Into<String>,
     source: impl Into<Vec<u8>>,
   ) {
-    if !self.0.contains_key(package_nv_ref.nv()) {
+    if !self.packages.contains_key(package_nv_ref.nv()) {
       self
-        .0
+        .packages
         .insert(package_nv_ref.nv().clone(), FromGraphNpmPackage::default());
     }
-    let package = self.0.get_mut(package_nv_ref.nv()).unwrap();
+    let package = self.packages.get_mut(package_nv_ref.nv()).unwrap();
```

Comment:
My comment about using `.entry` still stands :)

Reply:
oh, sure. I did this deliverately to avoid all the cloning, but not important anyway